kouncil-frontend/apps/kouncil/src/app/routing/routing.module.ts

```diff
@@ -93,13 +94,13 @@ const routes: Routes = [
       {path: 'login', component: LoginComponent},
       {path: 'changePassword', component: ChangePasswordComponent, canActivate: [AuthGuard]},
     ]
-  }
+  },
+  {path: 'oauth', component: OAuthRedirectComponent}
 ];
 
 @NgModule({
   imports: [
     RouterModule.forRoot(routes, {
-      useHash: true,
```

Comment:
why not?

Reply:
If you want to authenticate with github sso you have to pass callback url. If this url contains #, everything from this sign will be placed at the end of the response url